### PR TITLE
Dom api task

### PR DESCRIPTION
**Pull request by smillims**
### JS-DOM

[Demo](https://smillims.github.io/dom/) |
[Code base](https://github.com/smillims/dom)

The code is submitted in a dedicated feature branch.
Please, review.

### CODE REVIEW

**Review by babayK0 — CHANGES_REQUESTED** [review]
Nice work, but let improve your code together.
First of all maybe change function declarations to function expressions. With them your code will be more structured and clean.

Also see my comments below.

File: submissions/smillims/js-dom/script.js [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+		years: 57,
+		description: `Geralt of Rivia was a legendary witcher of the School of the Wolf active throughout the 13th century. He loved the sorceress Yennefer, considered the love of his life despite their tumultuous relationship, and became Ciri's adoptive father.`
+	},
+	{
+		id: 2,
+		name: "Ciri",
+		years: 20,
+		description: `Ciri. Also known as the Child of Destiny, Ciri is described generally as a weapon that has the potential to destroy the very world that she inhabits. Along with this there are ties to her being an apprentice to Geralt. Her Witcher training, coupled with the magical abilities, mark her as one of the most powerful individuals in the world.`
+	},
+	{	id: 3,
```
[the remark this answers is not in the document]
Comment (smillims) [reply submissions/smillims/js-dom/script.js:14]: Hey, I'm not in all understand what you are meaning

[the remark this answers is not in the document]
Comment (smillims) [reply submissions/smillims/js-dom/script.js:14]: oh, maybe I have already understood ^^

File: submissions/smillims/js-dom/script.js [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+function displayFact(navLink) {
+	const heroe = heroes.find( function (el) {
+ 		return navLink.dataset.id == el.id;
+ 	})
+ 	if (!heroe) {
+ 		return;
+ 	}
+	div.innerText = heroe.description;
+	container.append(div);
```
[the remark this answers is not in the document]
Comment (smillims) [reply submissions/smillims/js-dom/script.js:56]: I did in this way, but can't understand how to remove and add another `div` when i clicked on another item of my menu
```
const newDiv = document.createElement('div');
newDiv.classList.add('content-text');
newDiv.innerText = hero.description;
container.append(newDiv);
```

**Comment by smillims** [comment]
Deleted the local repo. Need to overwrite

**Pull request closed without merging**